src/index.js

```diff
@@ -0,0 +1,60 @@
+import './styles/style.css';
+import Alltask from './add.js';
+import { real, displayUI, newTask } from './reload.js';
+import { read, addButton } from './addtask.js';
+
+let tasks = [];
+addButton.addEventListener('click', (e) => {
+  tasks = JSON.parse(localStorage.getItem('list')) || [];
+  e.preventDefault();
+  const findIndex = tasks.length;
+  const task1 = new Alltask(read.value, false, findIndex + 1);
+  tasks.push(task1);
+  real(task1);
+  read.value = '';
+  localStorage.setItem('list', JSON.stringify(tasks));
```

Comment:
- DRY principle: I have noticed that you write that function in multiple places. I would suggest creating a separate function for setting an item to the local storage
- I would also suggest putting `'list'` local storage key into a const and setting it at the beginning of the file. 

Please checkout gist I created with index.js code with my updates:
https://gist.github.com/whid-bb/fc3932bee5c26c4db5501e1f4cfc5348